plugins/generator-1.16.4/forge-1.16.4/templates/json/biome.json.ftl

```diff
@@ -0,0 +1,251 @@
+{
+  "scale": ${data.heightVariation},
+  "effects": {
+    "mood_sound": {
+      "sound": "minecraft:ambient.cave",
+      "tick_delay": 6000,
+      "block_search_extent": 8,
+      "offset": 2.0
+    },
+    <#if data.foliafeColor?has_content>
+    "foliage_color": ${data.foliageColor.getRGB()},
+    <#else>
+    "foliage_color": 10387789,
+    </#if>
+    <#if data.grassColor?has_content>
+    "grass_color": ${data.grassColor.getRGB()},
+    <#else>
+    "grass_color": 9470285,
+    </#if>
+    <#if data.airColor?has_content>
+    "sky_color": ${data.airColor.getRGB()},
+    "fog_color": ${data.airColor.getRGB()},
+    <#else>
+    "sky_color": 7972607,
+    "fog_color": 12638463,
+    </#if>
+    <#if data.waterColor?has_content>
+    "water_color": ${data.waterColor.getRGB()},
+    <#else>
+    "water_color": 4159204,
+    </#if>
+    <#if data.waterFogColorColor?has_content>
+    "water_fog_color": ${data.waterColor.getRGB()}
+    <#else>
+    "water_fog_color": 329011
+    </#if>
+  },
+  "surface_builder": "${modid}:${registryname}",
+  "carvers": {
+    "air": [
+      "minecraft:cave",
+      "minecraft:canyon"
+    ]
+  },
+  "features": [
+    [],
+    [
+    "minecraft:lake_water",

```

Comment:
Lakes and such should probably be not hardcoded, or 1.16.4 biomes will not match the behavior of existing mods.

Reply:
I know, I just removed the old condition (of the template I made few weeks ago). I will change it when I will make default features. Don't worry ;)